src/client/App.jsx

```diff
@@ -1,9 +1,10 @@
 import './App.css';
 
 export default function App() {
-  return (
-    <div className="App">
 
-    </div>
-  );
+    return (
```

Comment:
What happened here?

Reply:
no idea, used the app for the tech skills call so might just be confused 
![image](https://user-images.githubusercontent.com/90713858/155174190-352ad5b1-7f2f-4f62-856e-e7b07ea6716c.png)